includes/PoolCounter.php

```diff
@@ -212,6 +214,7 @@ function execute( $skipcache = false ) {
 			default:
 				$errors = array( PoolCounter::QUEUE_FULL => 'pool-queuefull', PoolCounter::TIMEOUT => 'pool-timeout' );
 
+				$this->logError( $status );
```

Comment:
Yes, that's the main point of the change - I want to log the original status, hoping that I'll get information about the root cause. Does that make sense?